resources/views/exhibitions/index.blade.php

```diff
@@ -14,9 +14,8 @@
 @section('current')
     <div class="container-fluid py-3 bg-gdbo">
         <div class="container">
-            @if(!empty($current['data']))
-
-                <h3>Our current exhibitions</h3>
+            @if(!empty($current['data']) && !empty($displays['data']))
```

Comment:
This logic might be correct, but it seems unusual to me that both the arrays need to have data.

What if one array has data, and the other doesn't? It might lead to confusion for the client.

There might be a good reason for doing this that I'm not aware of though.

Reply:
Unfortunately I couldn't find free fontawesome svg that matches to current desing icon.

Reply:
Unfortunately I couldn't find free fontawesome svg that matches to current desing.